src/20_Components/amp-image-lightbox.html

```diff
@@ -23,45 +23,59 @@
     the lightbox element's ID.
   -->
   <amp-img on="tap:lightbox1"
-                role="button"
-                tabindex="0"
-                layout="responsive"
-                src="/img/Border_Collie.jpg" width="300" height="246"></amp-img>
+           role="button"
+           tabindex="0"
+           src="/img/Border_Collie.jpg"
+           alt="Picture of a dog"
+           title="Picture of a dog, view in lightbox"
+           layout="responsive"
+           width="300" height="246"></amp-img>
   <!-- This is the lightbox. The image shown in the lightbox does not need to be defined. -->
   <amp-image-lightbox id="lightbox1" layout="nodisplay"></amp-image-lightbox>
 
   <!--
     It is even possible to show different images in the same `amp-image-lightbox`. Here is another
-    image using the same lightbox. -->
+    image using the same lightbox.
+  -->
   <amp-img on="tap:lightbox1"
-      role="button"
-      tabindex="0"
-      layout="responsive"
-      src="/img/Hovawart.jpg" width="600" height="400"></amp-img>
+           role="button"
+           tabindex="0"
+           src="/img/Hovawart.jpg"
+           alt="Picture of a dog"
+           title="Picture of a dog, view in lightbox"
+           width="600" height="400"></amp-img>
 
   <!-- #### Captions -->
-   <!-- The amp-image-lightbox also can optionally display a caption for the image at the bottom
-     of the viewport. This can either be the contents of the `<figcaption>` element when the image
-     is in the figure tag...  -->
+  <!--
+    The amp-image-lightbox also can optionally display a caption for the image at the bottom
+    of the viewport. This can either be the contents of the `<figcaption>` element when the image
+    is in the figure tag...
+  -->
   <figure>
     <amp-img on="tap:lightbox1"
-    role="button"
-    tabindex="0"
-    src="/img/Border_Collie.jpg" width="300" height="246"></amp-img>
+             role="button"
+             tabindex="0"
+             src="/img/Border_Collie.jpg"
+             alt="Picture of a dog"
+             title="Picture of a dog, view in lightbox"
+             width="300" height="246"></amp-img>
     <figcaption>Border Collie.</figcaption>
   </figure>
 
   <!--
     ... or the contents of the element whose ID is specified by the image's
     <code>aria-describedby</code> attribute.
   -->
-  <amp-img on="tap:lightbox1"
-    role="button"
-    tabindex="0"
-    aria-describedby="imageDescription"
-    src="/img/Border_Collie.jpg" width="300" height="246"></amp-img>
-  <div id="imageDescription">
-    This is a border collie.
+  <div>
+    <amp-img on="tap:lightbox1"
+             role="button"
```

Comment:
I would still do `title` and `alt` similar to above here in addition to the `aria-describedby` which is providing additional information ( i.e. that it is a `border collie` dog)